samza-core/src/test/scala/org/apache/samza/metrics/reporter/TestJmxReporter.scala

```diff
@@ -63,7 +63,8 @@ object TestJmxReporter {
 class TestJmxReporter {
   import TestJmxReporter.url
 
-  @Test
+  // TODO: Fix in SAMZA-1194
```

Comment:
@navina 

Done both of your changes. 

https://issues.apache.org/jira/browse/SAMZA-1196 is created to track the fix. Not sure of the whom to assign, we can discuss it offline.